tests/Doctrine/Tests/DBAL/Functional/Schema/MySqlSchemaManagerTest.php

```diff
@@ -180,6 +180,21 @@ public function testDoesNotPropagateDefaultValuesForUnsupportedColumnTypes()
         $this->assertFalse($onlineTable->getColumn('def_blob_null')->getNotnull());
     }
 
+    public function testEscapedDefaultValueMustBePreserved()
```

Comment:
Since it's a functional test and it doesn't check any platform-specific logic, I think we could move it to a new `SchemaManagerTest` which instead of instantiating the schema manager would take it from `$conn->getSchemaManager()`.

This way, we could make sure this feature is portable and can be safely used on all supported platforms.

Reply:
I agree with you, but I don't think I'll be able to test al platforms unfortunately. I might give it a try via Travis though, but that might be cumbersome. I'll see what I can do...

Reply:
@PowerKiKi this is the right approach. You don't have to test other platforms yourself. I can run the updated tests on platforms not supported by Travis manually.